docs/sources/operations/storage/boltdb-shipper.md

```diff
@@ -84,6 +84,8 @@ For all the queries which require chunks to be read from the store, Queriers als
 
 ### Queriers
 
+**Note:** To avoid running Queriers as statefulset with persistent storage, we highly recommend running `IndexGateway` which takes care of downloading & syncing the index and serves it over gRPC to Queriers and Rulers.
```

Comment:
```suggestion
To avoid running Queriers as a StatefulSet with persistent storage, we recommend running an Index Gateway. An Index Gateway will download and synchronize the index, and it will serve it over gRPC to Queriers and Rulers.
```